light[] lights
float64 timestamp
sensor_msgs/Image img
